RevenueCatUI/CustomerCenter/Extensions/Store+Localized.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Store+Localized.swift
+//
+//
+//  Created by Facundo Menzella on 14/1/25.
+//
+import RevenueCat
+
+extension Store {
+
+    var localizedKey: CSCommonLocalizedString {
```

Comment:
Is this really localized though?

Reply:
It used to be `localizedName` but now it's just the localization key to access the translation.
I guess `localizationKey` would be a better name. 🤔

Reply:
Got it. Renamed `CSCommonLocalizedString` to `CCLocalizedString`. 🚀